functions/usage/index.js

```diff
@@ -167,14 +167,20 @@ exports.cloudifyUsage = function cloudifyUsage (req, res) {
             'cloudify_image_info': _getValue(row_data, 'metadata_image_info'),
             'cloudify_usage_tenants_count': _getValue(row_data, 'cloudify_usage_tenants_count'),
             'cloudify_usage_users_count': _getValue(row_data, 'cloudify_usage_users_count'),
+            'cloudify_usage_users_by_role': JSON.stringify(_getValue(row_data, 'cloudify_usage_users_by_role')),
```

Comment:
A general comment. This data is supposed to be used afterwards in the data studio. I wonder if it can be parsed as a string to extract the relevant information. If not, then these columns need to be of a different type.

Reply:
after consulting with @alexmolev, we've decided to send is as a dict for now